azure-dataplane-tests/src/main/java/com/azure/containers/containerregistry/implementation/models/AcrErrorInfo.java

```diff
@@ -102,7 +102,7 @@ public JsonWriter toJson(JsonWriter jsonWriter) throws IOException {
         jsonWriter.writeStartObject();
         jsonWriter.writeStringField("code", this.code);
         jsonWriter.writeStringField("message", this.message);
-        jsonWriter.writeUntyped(this.detail);
+        jsonWriter.writeUntypedField("detail", this.detail);
```

Comment:
This be the result of change on `Object` serialization. I think this be the correct form?